src/main/java/hudson/plugins/extendedread/SystemReadPermissionEnabler.java

```diff
@@ -0,0 +1,19 @@
+package hudson.plugins.extendedread;
+
+import hudson.Extension;
+import hudson.init.InitMilestone;
+import hudson.init.Initializer;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+@Extension
+@Restricted(NoExternalUse.class)
+public class SystemReadPermissionEnabler {
+
+    @Initializer(after = InitMilestone.PLUGINS_STARTED)
+    public static void enableSystemReadPermission() {
+        if (System.getProperty("jenkins.security.SystemReadPermission") == null) {
+            SystemReadPermission.SYSTEM_READ.setEnabled(true);
```

Comment:
NIT: In the current implementation it will be enabling ADMINISTER on older Jenkins core versions. I am not that concerned about that, but maybe it is an overkill